src/postcss/avatar-card.css

```diff
@@ -3,6 +3,8 @@ $avatar-name-line-height: $heading-line-height;
 $avatar-name-font-weight: $heading-font-weight;
 $avatar-name-font-color: $color-black-5;
 
+$small-size-multiplier: 0.7;
```

Comment:
I would rather suggest using fix values instead of multipliers for the sake of consistency.